clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/security/AbstractAtomicOperationsCredentialsConverter.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.security;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperationConverter;
+import com.netflix.spinnaker.credentials.CredentialsRepository;
+import com.netflix.spinnaker.kork.web.exceptions.InvalidRequestException;
+import java.util.stream.Collectors;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public abstract class AbstractAtomicOperationsCredentialsConverter<T extends AccountCredentials<?>>
+    implements AtomicOperationConverter {
+
+  @Autowired @Getter @Setter private CredentialsRepository<T> credentialsRepository;
+
+  @Getter private ObjectMapper objectMapper;
+
+  @Autowired
+  public void setObjectMapper(ObjectMapper objectMapper) {
+    // TODO(rz): This is a bad pattern, we should be using the object mapper customizer bean, rather
+    //  than modifying a singleton, global object mapper after injecting it somewhere.
+    this.objectMapper =
+        objectMapper
+            .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false)
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Can we take the opportunity to not propagate this pattern?

Reply:
Sure. Any reason you didn't just create a new objectMapper? These are in `@Component` so the instantiation cost is pretty negligible overall.

Reply:
Just used a new instance. The mapper is instantiated once per component.